.gitignore

```diff
@@ -128,3 +128,255 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
```

Comment:
[IMO] Nice template 👍

Reply:
Thanks,  this is autogenerated by the `.gitignore` extension 👍🏻